src/fireflyClient.ts

```diff
@@ -469,6 +422,17 @@ export class FireflyClient {
     const tokenContract = this.getContract(this._usdcToken, usdcContract);
     const marginBankContract = this.getContract(this._marginBank, mbContract);
 
+    //verify the user address via chainalysis
+    const verficationStatus = await this.verifyDeposit(amount);
+    if(verficationStatus.response.data.verificationStatus!== 'Success' && verficationStatus.response.data.verificationStatus!=null){
+        verficationStatus.ok = false;
+        verficationStatus.status = 5001;
+        verficationStatus.response.message=`Your address has been identified as ‘high risk’ by Chainalysis. 
```

Comment:
There's enum named `SuccessMessages`, which has contract errors strings, you can create similar one for this error